<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormAnnotationscaler</class>
 <widget class="QWidget" name="MainFormAnnotationscaler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>573</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Instructions:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1. Input the initial (current) scale of the annotations.&lt;/p&gt;&lt;p&gt;2. Input the final (desired) scale of the annotations.&lt;/p&gt;&lt;p&gt;3. Select the annotations to change the scale of on the right.&lt;/p&gt;&lt;p&gt;4. Click &lt;span style=&quot; font-style:italic;&quot;&gt;Process&lt;/span&gt; once.&lt;span style=&quot; font-weight:700;&quot;/&gt;If pressed too many times, click &lt;span style=&quot; font-style:italic;&quot;&gt;Reverse&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Initial Scale</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>X (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_initial_x"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Y (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_initial_y"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Z (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_initial_z"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Final Scale</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_final_x"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Y (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_final_y"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Z (nm):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_scale_final_z"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_process">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_reverse">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_output">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
